source/app/ppp/ui/widget_scroll_area.cpp

```diff
@@ -679,4 +696,23 @@ void CardScrollArea::showEvent(QShowEvent* event)
     setMinimumWidth(ComputeMinimumWidth());
 }
 
+void CardScrollArea::UpdateGlobalIncrementButton()
+{
+    if (m_GlobalIncrementButton == nullptr)
+        return;
+
+    // Check if any card is at the maximum value (100)
+    bool any_at_max = false;
+    for (const auto& [_, card] : m_Grid->GetCards())
+    {
+        if (m_Project.m_Data.m_Cards[card->GetCardName()].m_Num >= 100)
+        {
+            any_at_max = true;
+            break;
+        }
+    }
+
+    m_GlobalIncrementButton->setEnabled(!any_at_max);
```

Comment:
Shouldn't this be enabled until all cards reach 100?
Also this should be symmetric and disable the decrement button as well perhaps?

Forgot this comment earlier, sorry.